src/Model/Gpio.php

```diff
@@ -9,6 +9,9 @@
  */
 class Gpio
 {
+    const OUT	= 'OUT';
+    const IN	= 'IN';
+    const PWM	= 'PWM';
     const READ  = 'READ';
     const WRITE = 'WRITE';
 
```

Comment:
Read/Write: It might worth to have it's state ... On/Off 
$gpio->status and get On/Off